src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1059,6 +1075,27 @@ internal void DeleteModelInternal(List<ModelBase> modelsToDelete)
             OnDeletionComplete(this, EventArgs.Empty);
         }
 
+        internal void UngroupModel(List<ModelBase> modelsToUngroup)
```

Comment:
Hmm.. It feels like this behavior should be on the `Workspace`, not on `DynamoModel`.